monitor/__init__.py

```diff
@@ -0,0 +1,14 @@
+import os
+
+MONITOR_DIR = os.getcwd() + "/monitor/"
+MONITOR_CONF = "monitor.conf"
+
+MONITOR_DEFAULT_CONF = {
+    "ES_HOST": {"value": "", "type": str},
+    "ES_PORT": {"value": 9200, "type": int},
+    "KIBANA_HOST": {"value": "localhost", "type": str},
+    "KIBANA_PORT": {"value": "9243", "type": int},
+    "CLOUD_ID": {"value": "", "type": str}, 
+    "AUTH_USER": {"value": "user", "type": str},
+    "AUTH_PW": {"value": "password", "type": str},
+}
```

Comment:
Some defaults are missing here. Feel free to leave some empty if you feel like it (like remote auths), but one should at least be set so the code can run with defaults:

KIBANA_PORT should be `int` in defaults
CLOUD_ID has no default
ES_HOST has no default

Also, the default port pot Kibana is 5601 for me, isn't it for you?